### DISCUSSION TITLE
Seeking content for CS Analysis of Algorithms class

### DISCUSSION THREAD

**Discussion by dowobeha** [PrairieLearn/PrairieLearn#10588] [Course content sharing]
Hi, I'm teaching CS 411 (Analysis of Algorithms) at the University of Alaska Fairbanks. Would anyone happen to have any content for a similar class that they'd be willing to share?

**Comment by jeffgerickson** [comment]
I just invited you to the repository we use for CS 374 at Illinois (our required junior-level theory course).  See https://theorielearn.github.io/ for more info about our development project.  I'd be happy to give you staff access to the course on PrairieLearn and/or invite you to our group Slack channel; just let me know.  (But you can also find me on the PrairieLearn Slack.)

**Comment by dowobeha** [comment]
Thank you! Yes, that would be fantastic!
